hs-bindgen/src/HsBindgen/Errors.hs

```diff
@@ -0,0 +1,53 @@
+module HsBindgen.Errors (
+    HsBindgenException (..),
+    TODOException (..),
+    throw_TODO,
```

Comment:
it's `throw_TODO`, as there could be `throwIO_TODO`, which I haven't yet added as not needed.